src/gsy_e/gsy_e_core/util.py

```diff
@@ -443,21 +481,40 @@ def get_market_maker_rate_from_config(next_market, default_value=None, time_slot
                 raise e
         return find_object_of_same_weekday_and_time(GlobalConfig.market_maker_rate,
                                                     time_slot)
-    else:
-        return GlobalConfig.market_maker_rate
+    return GlobalConfig.market_maker_rate
+
+
+def get_feed_in_tariff_rate_from_config(next_market: "MarketBase",
+                                        default_value=None, time_slot=None):
```

Comment:
I just saw this: Here you set the default_value to None but before you return 0 if the time_slot is not found. Can you please explain?

Reply:
Took it as copy to `get_market_maker_rate_from_config`, it should either be some default or 0. In this case I will set it as zero